gsql/cache/cache_handler.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+from json.decoder import JSONDecodeError
+
+
+CacheDict = {}
+
+
+def keys_exists(element, *keys):
```

Comment:
Change `element` name to something more meaningful, could be `key_store`

Reply:
Done